### PR TITLE
[llvm-profgen] More tweaks to warnings

### PR DESCRIPTION
**Pull request by htyu**
Tweaking warnings more to avoid flooding user log.

### CODE REVIEW

File: llvm/tools/llvm-profgen/ProfiledBinary.cpp [hunk 1]
```diff
@@ -838,6 +849,11 @@ void ProfiledBinary::loadSymbolsFromDWARF(ObjectFile &Obj) {
   if (BinaryFunctions.empty())
     WithColor::warning() << "Loading of DWARF info completed, but no binary "
                             "functions have been retrieved.\n";
+  if (!AddrsWithMultipleSymbols.empty()) {
+    WithColor::warning() << "Found " << AddrsWithMultipleSymbols.size()
+                         << " start addresses have multiple symbols\n";
```
Comment (WenleiHe) [inline llvm/tools/llvm-profgen/ProfiledBinary.cpp:854]: nit: have -> with

File: llvm/tools/llvm-profgen/ProfiledBinary.h [hunk 2]
```diff
@@ -40,6 +40,7 @@
 #include <string>
 #include <unordered_map>
 #include <unordered_set>
+#include <utility>
```
Comment (WenleiHe) [inline llvm/tools/llvm-profgen/ProfiledBinary.h:43]: why is this change?

File: llvm/tools/llvm-profgen/ProfiledBinary.cpp [hunk 3]
```diff
@@ -707,6 +703,19 @@ void ProfiledBinary::disassemble(const ELFObjectFileBase *Obj) {
     }
   }
 
+  if (!AddrsWithInvalidInstruction.empty()) {
```
Comment (WenleiHe) [inline llvm/tools/llvm-profgen/ProfiledBinary.cpp:706]: It seems like we print aggregate after detailed warning for AddrsWithMultipleSymbols. Can we be consistent here?

File: llvm/tools/llvm-profgen/ProfiledBinary.h (hunk 2 above)
Comment (htyu) [reply llvm/tools/llvm-profgen/ProfiledBinary.h:43]: Good catch, it's not needed. Probably auto added as I previously used std::pair.

**Review by WenleiHe — APPROVED** [review]
lgtm, thanks.

**Pull request merged**